frontend/app/src/screens/object-items/object-items-paginated.tsx

```diff
@@ -65,7 +71,7 @@ export default function ObjectItems({
   const kindFilter = filters?.find((filter) => filter.name == "kind__value");
 
   if (schema && MENU_EXCLUDELIST.includes(schema.kind as string) && !preventBlock) {
-    return <Navigate to="/" />;
+    navigate("/");
```

Comment:
why ?

Reply:
why the other way around?

the component is using the useNavigate hook anyway, and it's meant to be used mainly if you can't use the navigate function for basic programmatic navigation

Reply:
it depends on the data flow.

In a react component, you must always return a JSX | null like:
```jsx
const Component = ({ isValid }) => {
  if (isValid) {
    return <Navigate to="/" />
  }
  
  return <Page />
}
```

Yes, the hook allows to do the same things, but im another a more declarative way:
```jsx
const Component = ({ isValid }) => {  
  return <Button onClick={() => navigate("/")} />
}
```
or
```jsx
const Component = ({ isValid }) => {  
  useEffect(() => {
  if (!isValid) navigate(/"")
}, [isValid])
  return <Page />
}
```